src/Symfony/Component/Validator/README.md

```diff
@@ -7,7 +7,7 @@ The Validator component provides tools to validate values following the
 Resources
 ---------
 
-  * [Documentation](https://symfony.com/doc/current/book/validation.html)
+  * [Documentation](https://symfony.com/doc/current/components/validator.html)
```

Comment:
@xabbuh the problem is that I want to use the "component" no the symfony integration :)